pkg/proxy/apis/kubeproxyconfig/v1alpha1/types.go

```diff
@@ -65,7 +65,7 @@ type KubeProxyIPVSConfiguration struct {
 	Scheduler string `json:"scheduler"`
 	// excludeCIDRs is a list of CIDR's which the ipvs proxier should not touch
 	// when cleaning up ipvs services.
-	ExcludeCIDRs []string
+	ExcludeCIDRs []string `json:"excludeCIDRs"`
```

Comment:
omitempty?

Reply:
This can be a follow up for the proxy team. Since this isn't a served API, the distinction is not super important, and if we add omitempty later, it doesn't affect how the config is read

Reply:
@thockin ^

Reply:
I needed a squash, so added the "omitempty".

Reply:
Removed. Broke a bunch of tests.